### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Arafatk**
@pushrax  Maybe add a Circle CI Badge too like [here](https://github.com/Arafatk/liquid).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * [Version history](History.md)
 * [Liquid documentation from Shopify](http://docs.shopify.com/themes/liquid-basics)
 * [Liquid Wiki at GitHub](https://github.com/Shopify/liquid/wiki)
+* [Liquid Cheet Sheet](http://cheat.markdunkley.com/)
```
Comment (dylanahsmith) [inline README.md]: Most of that is Shopify specific, which could cause confusion in the context of liquid being used as a gem.

Comment (dylanahsmith) [reply README.md]: I guess the same could be said about the objects documented in the "Liquid documentation from Shopify" link, but at least that one is clearer from the link text that it is referring to shopify's liquid documentation.

Comment (Arafatk) [reply README.md]: @dylanahsmith Yup!!    
Well that kinda makes sense.     
Anyways, you can add the Circleci badge if that's okay.    
Do you mind if I close this pr?

Comment (dylanahsmith) [reply README.md]: I don't even know why we have both CircleCI and TravisCI.

I don't mind if you close the PR.

**Pull request closed without merging**